ui/src/scss/_patterns_navigation.scss

```diff
@@ -15,4 +15,8 @@
       display: none;
     }
   }
+
+  .p-navigation__maas-logo {
+    vertical-align: top;
+  }
```

Comment:
I don't think this is necessary with the old markup

Reply:
Yep you're right!